src/components/message/index.jsx

```diff
@@ -12,12 +12,13 @@ export const Message = (props) => {
       <div className={styles["name"]} style={!type ? { color: color } : {}}>
         {name}, <span>{time}</span>
       </div>
-      <div className={styles["chat-bubble"]}>
+      <div className={styles["chat-bubble"]}
+       style={err?{color: "#7a7a7a"}:{}}>
         <div
           className={styles["colorBg"]}
           style={!type ? { backgroundColor: color } : {}}
         />
-        {message}
+        {message + (err?"  ( Not sent )":"")}
```

Comment:
message display of error [#167](https://github.com/webaverse-studios/CharacterCreator/issues/167)